app/adapters/certificate.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationAdapter from './application';
+import buildURLWithPrefixMap from '../utils/build-url-with-prefix-map';
+
+export default ApplicationAdapter.extend({
+  buildURL: buildURLWithPrefixMap({
```

Comment:
This file seems identical to the app adapter. Can we either make a generic name for this and use that for these two files, or just have this file be `export { default } from './app';`?